impl/pom.xml

```diff
@@ -44,6 +44,10 @@
             <groupId>com.okta.commons</groupId>
             <artifactId>okta-http-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.okta.commons</groupId>
+            <artifactId>okta-http-okhttp</artifactId>
```

Comment:
This dep should be scoped to `test` as you should not be compiling against it directly

Reply:
Removed this dep.

Reply:
This needs to be scoped as `test`, there should not be a direct dependency here